srcpkgs/python3-yapf/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'python3-yapf'
 pkgname=python3-yapf
-version=0.33.0
-revision=2
+version=0.40.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-tomli"
+depends="python3-importlib_metadata python3-platformdirs python3-tomli"
+checkdepends="${depends} python3-pytest"
 short_desc="Formatter for Python code"
 maintainer="RinsedSloth <afernandezh@protonmail.com>"
```

Comment:
are you interested in adopting this? current maintainer hasn't touched it since addition and you've done all the updates

Reply:
thanks, pushed the changes